tests/test_booster/test_plugin/test_3d_plugin.py

```diff
@@ -85,10 +112,149 @@ def check_3d_plugin(init_method: str = "none", early_stop: bool = True):
     assert len(failed_info) == 0, "\n".join([f"{k}: {v}" for k, v in failed_info.items()])
 
 
+@parameterize(
+    "test_args",
+    [
+        {
+            "batch_size": 2,
+            "num_steps": 8,
+            "tp": 2,
+            "pp": 2,
+            "pp_style": "1f1b",
+            "num_model_chunks": 1,
+            "mbs": 2,
+            "zero": 0,
+            "precision": "fp16",
+            "initial_scale": 1,
+            "max_length": 512,
+            "gradient_accumulation_step": 4,
+        },
+        {
+            "batch_size": 1,
+            "num_steps": 8,
+            "tp": 1,
+            "pp": 2,
+            "pp_style": "1f1b",
+            "num_model_chunks": 1,
+            "mbs": 1,
+            "zero": 1,
+            "precision": "fp16",
+            "initial_scale": 1,
+            "max_length": 512,
+            "gradient_accumulation_step": 4,
+        },
+        {
+            "batch_size": 1,
+            "num_steps": 8,
+            "tp": 2,
+            "pp": 1,
+            "pp_style": "1f1b",
+            "num_model_chunks": 1,
+            "mbs": 1,
+            "zero": 2,
+            "precision": "fp16",
+            "initial_scale": 1,
+            "max_length": 512,
+            "gradient_accumulation_step": 4,
+        },
+        {
+            "batch_size": 1,
+            "num_steps": 8,
+            "tp": 2,
+            "pp": 1,
+            "pp_style": "1f1b",
+            "num_model_chunks": 1,
+            "mbs": 1,
+            "zero": 0,
+            "precision": "fp16",
+            "initial_scale": 1,
+            "max_length": 512,
+            "gradient_accumulation_step": 4,
+        },
+    ],
+)
+def run_grad_acc_test(test_args):
+    _, (model_fn, _, _, _, _) = list(model_zoo.get_sub_registry("transformers_gpt_lm").items())[0]
```

Comment:
If you only want to test one model, you should use a more concrete name (not just "transformers_gpt_lm")